src/forms/FormFieldChipSingleSelect/FormFieldChipSingleSelect.stories.tsx

```diff
@@ -131,7 +130,7 @@ export const KitchenSink = (): ReactElement => {
 					helperText: "Helper text",
 					instructionText: "Instruction text"
 				}
-			] as FieldDef<FormFieldChipSingleSelectDef>[],
+			] as FieldDef[],

```

Comment:
For consistency please move the "as FieldDef[]"s next to their respective "const fields"